src/org/opendatakit/briefcase/export/ExportConfiguration.java

```diff
@@ -174,8 +173,7 @@ public ExportConfiguration setExportDir(Path path) {
   }
 
   public ExportConfiguration setPemFile(Path path) {
-    //Check if path is a valid pem file with privateKey
-    this.pemFile = Optional.ofNullable(path).filter(PrivateKeyUtils::isValidPrivateKey);
+    this.pemFile = Optional.ofNullable(path);
```

Comment:
Are you sure about this? We talked about how we want this class to behave like a Value Object and how important is to validate any value we want to set... I think you shouldn't remove that validation from here. If you did this because you had trouble in other parts, you need to fix the other parts, not this one...